modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/PodcastSettingsFragment.kt

```diff
@@ -490,6 +517,18 @@ class PodcastSettingsFragment : BasePreferenceFragment(), FilterSelectFragment.L
         preferenceFilters?.isVisible = availableFilters.isNotEmpty()
     }
 
+    private fun updateAutoDownloadLimit(podcast: Podcast) {
+        val options = AutoDownloadLimitSetting.entries
+        val podcastAutoDownloadLimit = podcast.autoDownloadLimit ?: AutoDownloadLimitSetting.TWO_LATEST_EPISODE
```

Comment:
Nothing to change here right now but it will be important to remember to use `TWO_LATEST_EPISODE` as the default value when doing actual downloading.